docs/file_formats/fip.md

```diff
@@ -8,6 +8,28 @@
 
 This document describes the standards for the acquisition of frame-projected independent-fiber photometry (FIP) data in the Allen Institute for Neural Dynamics.
 
+## Terminology
+
+This document uses specific terminology to distinguish between different components of the FIP system:
+
+* **Fibers**: Optical fibers surgically implanted in the mouse's brain. A mouse may have 1-4 implanted fibers. Fiber implantation details (locations, stereotactic coordinates) are documented in the procedures.json metadata file.
+
+* **Patch cords**: Fiber optic cables that are permanently part of the FIP hardware system. The current FIP system has exactly 4 patch cords. At experiment time, experimenters physically connect patch cords to implanted fibers. Not all patch cords are necessarily connected during every experiment.
+
+* **Fiber bundle**: The assembly of all patch cords whose termini are visible to the imaging cameras. The fiber bundle remains stationary during experiments.
```

Comment:
I dont think the file format should specify this. Nothing is preventing the file format from accommodating multiple bundles per acquisition OR even moving bundles. In fact, we have previously floated the idea of connecting a rotary joint directly to the focusing element for freely moving experiments.